fusion_toolbox/reaction_rate/bosch_hale.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+
```

Comment:
there should be 2 line spacing for module level classes and functions and single line spacing within functions and classes per PEP8

Reply:
Fixed with 6e8181d20090b5ee6cf75efc2ee198de7d8cc5f7